roomCodechanges (#1471)
100mslive/100ms-docs: docs/concepts/v2/concepts/security-and-tokens.mdx

100ms client SDKs use auth tokens to [join a room](/javascript/v2/how--to-guides/set-up-video-conferencing/join).

<FlexContainer>
<Callout icon="shield" title="How to get auth tokens?">

> 1. No-Code way to get started - Get temporary auth tokens from [the dashboard](/javascript/v2/guides/token#get-a-temporary-token-from-100ms-dashboard)

Reviewer: let's plan to move this doc to the concepts page - it doesn't need to be inside a platform and aligns with your thinking on having "docs for dashboard"

Author: taking this up in a separate PR

2. [Auth token server](#set-up-your-token-server) - Programmatically generate auth tokens by setting up an auth token server

> 3. Via **[Room Codes](/concepts/v2/concepts/rooms/room-codes/room-codes)** - Simply get auth tokens by generating room codes using server-side [Create Room Code API](/server-side/v2/api-reference/room-codes/create-room-code-api) . This approach eliminates the need to set up an auth token server at your end. Head to [Room Code Authentication](/concepts/v2/concepts/rooms/room-codes/room-code-auth) for more details.

Reviewer: i think this sentence can be shortened and have 1 clear link as CTA (not 2)

```suggestion
3. Via **[Room Codes](/concepts/v2/concepts/rooms/room-codes/room-codes)** - Get auth tokens by generating unique room codes. Head to [Room Code Authentication](/concepts/v2/concepts/rooms/room-codes/room-code-auth) for more details.

```
